systemtest/src/test/java/io/strimzi/systemtest/operators/FeatureGatesST.java

```diff
@@ -235,23 +233,22 @@ void testSwitchingConnectStabilityIdentifiesFeatureGateOnAndOff(ExtensionContext
         final LabelSelector connectLabelSelector = KafkaConnectResource.getLabelSelector(testStorage.getClusterName(), KafkaConnectResources.deploymentName(testStorage.getClusterName()));
         Map<String, String> connectPods = PodUtils.podSnapshot(clusterOperator.getDeploymentNamespace(), connectLabelSelector);
 
+        String startingConnectorPodName = kubeClient().listPods(testStorage.getNamespaceName(), connectLabelSelector).get(0).getMetadata().getName();
+
         KafkaClients clients = new KafkaClientsBuilder()
-            .withProducerName(testStorage.getProducerName())
+            .withProducerName("starting-fg-producer")
             .withConsumerName(testStorage.getConsumerName())
             .withBootstrapAddress(KafkaResources.plainBootstrapAddress(testStorage.getClusterName()))
             .withTopicName(testStorage.getTopicName())
-            .withMessageCount(messageCount)
-            .withDelayMs(500)
+            .withMessageCount(40)
```

Comment:
Having this in variable and then doing something like this

```
KafkaConnectUtils.waitForMessagesInKafkaConnectFileSink(testStorage.getNamespaceName(), startingConnectorPodName, Constants.DEFAULT_SINK_FILE_PATH, String.join("Hello-world - %s", messageCount));
```

would make it easier to edit the test if someone would decide to change the message count.

Then you can update it with new message count on other places as well.